python/Ganga/Lib/LCG/Grid.py

```diff
@@ -188,23 +149,14 @@ def __resolve_no_matching_jobs__(cmd_output):
     return glite_ids
 
 
-def list_match(jdlpath, ce=None):
+def list_match(jdlpath, cred_req, ce=None):
```

Comment:
Across the codebase, `cred_req` is used to designate a `ICredentialRequirement` object being passed around. I can go through `Grid.py` and add in docstrings explaining this.